plotly/add_figure_factory_to_plotly_library.md

```diff
@@ -0,0 +1,165 @@
+# Add A Figure Factory to the Plotly [Python Library](https://plot.ly/python/)
+
+## What is a Figure Factory?
+In the Python Plotly Library, we have very basic plot types that are created using the `plotly.graph_objs` module. These plot types include Scatter, Box and Bar types. They are the basis of the plots and charts instatiated by Plotly.
```

Comment:
- we have basic plot types that are created using the `plotly.graph_objs` module
- These plot types include Scatter, Box and Bar types. - you could link to the graph_objs file where it lists all the plot types